app/models/manageiq/providers/openstack/cloud_manager/refresh_parser.rb

```diff
@@ -271,13 +271,19 @@ def parse_host_aggregate(ha)
     def parse_tenant(tenant)
       uid = tenant.id
 
+      flavor_ids = []
+      @ems.with_provider_connection(:tenant => tenant.id) do |connection|
+        flavor_ids = connection.handled_list(:flavors).map(&:id)
```

Comment:
we should also add the column publicly_available for flavor, so we are consistent with the same membership pattern for images